src/git-duet/git-duet-cherry-pick/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("cherry-pick"))
```

Comment:
After a little more thought, I think the `--signoff` on `cherry-pick` was probably meant to sign off on the commits you are cherry picking (that previously only had an author). By setting this here, it would override any committer present on the original commit -- it feels like this might be desirable, but definitely deserves a second thought.